### DISCUSSION TITLE
KEDA autoscaler causing issues with retrieving logs

### DISCUSSION THREAD

**Discussion by valkmit** [apache/airflow#28286] [Q&A] [answered]
Hi team,

I have an airflow installation deployed to a Kubernetes cluster, where the workers are automatically scaled with KEDA. I have the minimum worker count set to 0, so my cluster is allowed to spin down all worker pods.

This seems to create a bit of a problem with log files, though. While the pod is active, I'm able to view the log output in the web UI - but after KEDA has terminated the pod, the logs are no longer visible. I was under the impression that logs were persisted onto a PVC?

What is the recommended approach for this problem? Simply raising the minimum worker count to 1, so that at least 1 worker always exists in the cluster? This is slightly suboptiomal for me as I would prefer workers be spun down when not in use so that their filesystem gets cleared every so often.

For reference, I am using the provided helm chart for installation.

**Comment by potiuk** [comment, marked as answer]
Use remote logging - with GCP/S3/Elasticsearch - whatever suits you https://airflow.apache.org/docs/apache-airflow-providers/core-extensions/logging.html
